packages/react/src/components/Input/index.js

```diff
@@ -50,6 +53,10 @@ const Input = React.forwardRef(
         {isValid && (
           <Icon icon={Icon.ICONS.IconCheck} className='f-FormEl-icon' />
         )}
+
+        {!isValid && !isInvalid && icon && (
```

Comment:
Does this actually work?
Shouldn't it be 
```suggestion
        {!isValid || !isInvalid || icon || (
```

Reply:
Nevermind 🤦‍♂ 
Should we export a function to get only 1 icon and clear this code a bit?
Like
```
const getIcon = () => {
  if (isInvalid) return '...'
  if (isValid) return '...'
  return icon
}
```
WDYT?

Reply:
I don't get this code, am I missing something? `!isValid && !isInvalid`, isn't that the same as a single `!isValid`?

Reply:
🤦‍♂ ok, now I see they're two different vars. Maybe consider changing the name? Two people got facepalmed with this haha. I agree with Ugo's proposal